src/components/FallbackImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+import { Image, ImageProps } from 'antd'
+
+interface FallbackImageProps {
+  src: string | undefined
+  fallbackSrc: string | undefined
+  rest: ImageProps
+}
+
+const FallbackImage = ({ src, fallbackSrc, rest }: FallbackImageProps) => {
```

Comment:
**suggestion** could we rename `rest` to `...props`? That way we don't need to pass in `rest` prop: can just pass all the props along side `src` and `fallbackSrc`

Reply:
using next image component fallback instead